crawl-ref/source/spl-data.h

```diff
@@ -2201,7 +2157,7 @@ static const struct spell_desc spelldata[] =
 
 {
     SPELL_INJURY_BOND, "Injury Bond",
-    spschool::charms,
+    spschool::hexes,
```

Comment:
or maybe necro?

Reply:
Picked this because Summon Guardian Golem is Summon/Hex